server/storage/mysql_test.go

```diff
@@ -22,15 +22,15 @@ func init() {
 		logrus.Fatal("MYSQL environment variable not set")
 	}
 
-	for i := 0; i < 30; i++ {
+	for i := 0; i <= 30; i++ {
 		gormDB, err := gorm.Open("mysql", dburl)
 		if err == nil {
 			err := gormDB.DB().Ping()
 			if err == nil {
 				break
 			}
 		}
-		if i == 29 {
+		if i == 30 {
```

Comment:
@endophage you are right about the iteration, no matter using `30` or `29`, I think it is used for line 34 to log out how much time has been past.

Because of the `sleep statement`  is at the end of this `for` loop, so when `i == 30` and the process reaches line 34 to log out sth, we may have just sleep 30 times, thus 60 seconds.